src/spectrecoff-cli/Program.fs

```diff
@@ -234,6 +234,16 @@ let main argv =
         config
             .AddCommand<Progress>("progress")
             .WithDescription("Shows which modules from Spectre.Console have been ported to SpectreCoff.")
-            |> ignore)
+            |> ignore
+
+        config.AddBranch("theme", fun(add: IConfigurator<ThemeSettings>) ->
```

Comment:
Did you take my branch, and rewrite everything into one commit? Appropriate all the hard work :P ?

Reply:
What branch? I did all of this myself!

Nah, I appropriated yours of course, but the commit history was dirty and I had to rebase, so I needed to force push a little, which I can only do in my own branch. 

So, in summary: It is all mine now.